sktime/forecasting/rbf_forecaster.py

```diff
@@ -0,0 +1,601 @@
+"""RBF Neural Networks for Time Series Forecasting."""
+
+__author__ = ["phoeenniixx"]
+
+import numpy as np
+import pandas as pd
+from sklearn.preprocessing import StandardScaler
+
+from sktime.forecasting.base import BaseForecaster
+from sktime.transformations.series.basisfunction import RBFTransformer
+from sktime.utils.dependencies._dependencies import _check_soft_dependencies
+
+_check_soft_dependencies("torch", severity="warning")
+
+
+class RBFForecaster(BaseForecaster):
+    r"""Forecasting model using RBF transformations and 'NN' layers for time series.
+
+    This forecaster uses an RBF layer to transform input time series data into a
+    higher-dimensional space, which is then used by neural network layers for
+    forecasting.
+
+    Parameters
+    ----------
+    window_length : int
+        Length of the input sequence for each window.
+    hidden_size : int, optional (default=32)
+        Number of units in the RBF layer.
+    batch_size : int, optional (default=32)
+        Size of mini-batches for training.
+    centers : array-like, optional (default=None)
+        Center points for RBF transformations.
+    gamma : float, optional (default=1.0)
+        Scaling factor controlling the spread of the RBF.
+    rbf_type : {"gaussian", "multiquadric", "inverse_multiquadric"}
+                optional (default="gaussian")
```

Comment:
this might not display correctly in the docstring, have you checked the readthedocs build? ("details" next to the readthedocs CI element)

Reply:
The doc-build is not showing any error and when i tried the "details" part, I gets redirected to this site: https://sktime--7334.org.readthedocs.build/en/7334/

So to see any errors I need to register there?
I have done that as well... 
maybe after I get registered as a user I will be able to see any issue?